src/layout/IHyprLayout.hpp

```diff
@@ -206,4 +214,6 @@ class IHyprLayout {
     eRectCorner  m_eGrabbedCorner = CORNER_TOPLEFT;
 
     PHLWINDOWREF m_pLastTiledWindow;
+
+    void         performSnap(Vector2D& pos, Vector2D& size, PHLWINDOW DRAGGINGWINDOW, const eMouseBindMode mode);
```

Comment:
at this point, does it need to be a member? Can we just make it `static` in .cpp?